lib/vagrant-service-manager/service.rb

```diff
@@ -0,0 +1,35 @@
+# Loads all services
+Dir["#{File.dirname(__FILE__)}/services/*.rb"].each { |f| require_relative f }
+
+module Vagrant
+  module ServiceManager
+    SUPPORTED_BOXES = ['adb', 'cdk']
+
+    class Service
+      def initialize(app, env)
+        @app = app
+        @machine = env[:machine]
+        @ui = env[:ui]
+        @services = @machine.config.servicemanager.services.split(',').map(&:strip)
+        @docker_hook = Docker.new(@machine, @ui)
+        @openshift_hook = OpenShift.new(@machine, @ui)
+      end
+
+      def call(env)
+        @app.call(env)
+
+        if SUPPORTED_BOXES.include? @machine.guest.capability(:os_variant)
```

Comment:
@bexelbie : We are doing the same thing here. In line no:6 of this file we are defining this
```
SUPPORTED_BOXES = ['adb', 'cdk']
```
and the guest is checked if it is subset of `SUPPORTED_BOXES`.

Did I answer your question or you wanted to ask something else?